app/pods/publications/publication/proofs/index/controller.js

```diff
@@ -88,6 +88,23 @@ export default class PublicationsPublicationProofsController extends Controller
     this.showProofRequestModal = false;
   }
 
+  @task
+  *deleteReceivedPiece(proofReceivedEvent, piece) {
+    yield this.performDeleteReceivedPiece(proofReceivedEvent, piece);
+    this.send('refresh');
```

Comment:
Is deze refresh noodzakelijk? Er worden geen activities verwijderd, enkel mogelijks een end-date aangepast. Aangezien er gebruik gemaakt wordt van `get isShown()` in de `TimelineActivity` lijkt de view mij correct te updaten?